content/pages/manual/devices/vuams.md

```diff
@@ -1,8 +1,102 @@
+---
+layout: osdoc
 title: Ambulatory Monitoring System (VU-AMS)
+group: Devices
+permalink: /vu-ams/
+---
 
-The VU University Ambulatory Monitoring System (VU-AMS) is a device that can be used to measure a variety of factors related to heart rate, respiration, and body movement. The developers offer an OpenSesame template on their website.
+The VU University Ambulatory Monitoring System (VU-AMS) is a device that can be used to measure a variety of factors related to heart rate, respiration, and body movement. The developers made an OpenSesame [plugin](https://github.com/JdenHartog/vu_ams).
 
-For more information, see:
+- <http://www.vu-ams.nl/>
+
+%--
+figure:
+ id: FigVuamsDevice
+ source: VU-AMS.png
+ caption: |
+  The VU-AMS device.
+--%
+
+[TOC]
+
+## Using the `vu-ams` plugin
+
+Parallel_port_trigger is a third-party plugin.
```

Comment:
Oops.... 
It clearly shows I started out with the Parallel_port_trigger plugin as a template :innocent: